packages/utils/index.ts

```diff
@@ -36,10 +39,51 @@ export const bufferToJSON = (metadata: Uint8Array) => {
   return JSON.parse(new TextDecoder().decode(metadata));
 };
 
+export const decodeBuffer = (buffer: Uint8Array) => {
+  const textDecoder = new TextDecoder();
+  return textDecoder.decode(buffer);
+};
+
 export const stringifyToBuffer = (data: any) => {
   return new TextEncoder().encode(JSON.stringify(data));
 };
 
-export const zeroAddress = "0x0000000000000000000000000000000000000000";
-export const anvilAddress =
+export function priceToUint256(priceString: string, decimals = 18) {
+  // Parse the price string to a bigint
+  const priceInSmallestUnit = parseUnits(priceString, decimals);
+  // Convert bigint to 32 byte directly
+  const buffer = numberToBytes(priceInSmallestUnit, { size: 32 });
+  return buffer;
+}
+export function formatPrice(price: string, decimal: number) {
+  return formatUnits(BigInt(price), decimal);
+}
```

Comment:
Please add a comment to this function describing why it should be use over `formatUnits`. Also this looks like it will format things other then prices so maybe it should be named differently.